src/mantarray_desktop_app/mc_simulator.py

```diff
@@ -131,46 +145,80 @@ def _send_data_packet(
 
     def _commands_for_each_run_iteration(self) -> None:
         self._handle_test_comm()
+        # if _reboot_time_secs is not None, this means the simulator is in a "reboot" phase
+        if self._reboot_time_secs is not None:
+            secs_since_reboot = _get_secs_since_reboot_command(self._reboot_time_secs)
+            # if secs_since_reboot is less than the reboot duration, simulator is still in the 'reboot' phase. Commands from PC will be discared and status beacons will not be sent
```

Comment:
would commands from PC actually be discarded?  or would they just wait in the USB wire until the Mantarray rebooted again?  Was this something you checked with Aster? I don't remember explicitly discussing this, but I may have forgotten

Reply:
Aster said she was unsure if the UART buffer is active during reboot but if it is we should assume for now that it will get flushed after rebooting and the command will never be seen